### PR TITLE
Adding geodemora recipe

### PR DESCRIPTION
**Pull request by ramendra1990**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ramendra1990** [comment]
@conda-forge/help-python the recipe has passed all the checks and is ready for your review. Thanks.

File: recipes/geodemora/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - python >=3.8
+  run:
+    - folium
+    - ipyleaflet
+    - pyshp
+    - python >=3.8
+
+test:
+  imports:
+    - geodemora
```
Comment (BastianZim) [inline recipes/geodemora/meta.yaml:29]:
```suggestion
test:
  imports:
    - geodemora
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/geodemora/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```
Comment (BastianZim) [inline recipes/geodemora/meta.yaml:20]:
```suggestion
    - python >=3.7
```
Ref: https://github.com/ramendra1990/geodemora/blob/2cee3b7398f95baa4433c03573f5b58879ef1d58/setup.py#L30

File: recipes/geodemora/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - folium
+    - ipyleaflet
+    - pyshp
+    - python >=3.8
```
Comment (BastianZim) [inline recipes/geodemora/meta.yaml:25]:
```suggestion
    - python >=3.7
```

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

**Pull request merged**